src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +124,20 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
+     * Sorts all persons in the list
+     * Modifies internalList
+     */
+    public void sorted() {
+        // internal list is a list of persons
+        internalList.sort(new Comparator<Person>() {
```

Comment:
Since `Person` already implements `Comparable`, you can simply do `Collections.sort(internalList)`